test/unit/get-default-blueprint-name-override-test.js

```diff
@@ -45,4 +45,9 @@ describe(getBlueprintNameOverride, function() {
     let defaultBlueprintOverride = await getBlueprintNameOverride(localPackageFixture);
     expect(defaultBlueprintOverride).to.be.equal('custom-blueprint');
   });
+
+  it('NPM package with nondefault returns expected value', async function() {
```

Comment:
nitpick but this test should be moved to a new file test/integration/get-default-blueprint-name-override-test.js

Reply:
as well as the above one which is testing the disk blueprints

Reply:
done!